install_config/install/advanced_install.adoc

```diff
@@ -1181,6 +1181,139 @@ xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#i
 openshift_hosted_logging_storage_kind=dynamic
 ----
 
+[[enabling-service-catalog]]
+=== Enabling the Service Catalog
+
+[NOTE]
+====
+The service catalog is currently a Technology Preview feature.
+====
```

Comment:
There's efforts around the consistency of tech preview features. There's an example here: https://docs.openshift.com/container-platform/3.5/admin_guide/managing_networking.html#admin-guide-networking-networkpolicy